test_harness/src/lib.rs

```diff
@@ -977,15 +993,25 @@ pub mod run {
             .await
             .expect("Should flush redis database");
 
+        let logger = new_logger(&validator.sentry_logger_prefix);
         let campaign_remaining = CampaignRemaining::new(redis.clone());
 
+        // todo: Make platform_url configurable! Load from config or pass with env. variable
+        let platform_url = "https://todo-local-platform"
+            .parse()
+            .expect("Bad ApiUrl, load from Config?");
+        // todo: Make keep_alive_interval configurable!
+        let platform_api = PlatformApi::new(platform_url, std::time::Duration::from_secs(3))
+            .expect("Should make PlatformApi");
```

Comment:
```suggestion
       let platform_api = PlatformApi::new(
          config.platform.url.clone(),
          config.platform.keep_alive_interval,
      )
      .expect("Failed to build PlatformApi");
```

We can use the values from config here as well.